templates/vue/src/components/tyde/TydeStage.vue

```diff
@@ -74,6 +80,10 @@ export default {
   --tyde-green: #205a27;
   --tyde-border-green: #39b54a;
 }
+
+.stage-open {
```

Comment:
To avoid conflicts, please add `tapestry` at the beginning of your class name. i.e. `tapestry-stage-open` instead of `stage-open`. Even better, put `body.tapestry-stage-open` instead so it only applies to the body.